src/components/academyComponent/HandsOnCourses.jsx

```diff
@@ -0,0 +1,106 @@
+
+
+const HandsOnCourses = () => {
+  const containerStyle = {
+    padding: '4rem 1rem',
+    textAlign: 'center',
+    backgroundColor: "#e5e5e5",
+    fontFamily: "monospace"
+  };
+
+  const headingStyle = {
+    color: '#8A4DFF',
+    fontSize: '2.5rem',
+    fontFamily: 'monospace',
+    marginBottom: '3rem',
+  };
+
+  const coursesContainerStyle = {
+    display: 'flex',
+    justifyContent: 'center',
+    gap: '2rem',
+    flexWrap: 'wrap',
+    marginBottom: '2rem',
+  };
+
+  const cardStyle = {
+    backgroundColor: '#fff',
+    padding: '2rem',
+    borderRadius: '1.5rem',
+    boxShadow: '0 4px 8px rgba(0, 0, 0, 0.1)',
+    width: '300px',
+    textAlign: 'center',
+  };
+
+  const iconStyle = {
+    fontSize: '3rem',
+    color: '#8A4DFF',
+    marginBottom: '1rem',
+  };
+
+  const titleStyle = {
+    fontFamily: 'Roboto',
+    fontSize: '1.5rem',
+    fontWeight: 'bold',
+    color: 'black',
+    margin: '0.5rem 0',
+  };
+
+  const descriptionStyle = {
+    fontSize: '0.9rem',
+    color: '#444',
+    marginBottom: '1.5rem',
+  };
+
+  const buttonStyle = {
+    backgroundColor: '#8A4DFF',
+    color: '#fff',
+    padding: '0.75rem 1.25rem',
+    borderRadius: '0.5rem',
+    fontWeight: 'bold',
+    border: 'none',
+    cursor: 'pointer',
+  };
+
+  return (
+    <div style={containerStyle}>
+      <h2 style={headingStyle}>HANDS-ON COURSES</h2>
+
+      <div style={coursesContainerStyle}>
+        {/* Card 1 */}
+        <div style={cardStyle}>
+          <div style={iconStyle}>🧩</div>
```

Comment:
This icon has nothing to do with API development.
I'll suggest using a small image. You can find good ones online